File: src/com/eagle/programmar/CMacro/Terminals/CMacro_PunctuationChoice.java
// Copyright Eagle Legacy Modernization, LLC, 2010-date
// Original author: Steven A. O'Hara, Dec 30, 2020

package com.eagle.programmar.CMacro.Terminals;

import com.eagle.tokens.terminals.TerminalPunctuationChoice;

public class CMacro_PunctuationChoice extends TerminalPunctuationChoice
{
	// Need default constructor for reading from the XML file
	public CMacro_PunctuationChoice()
	{
		super();
	}
	
	public CMacro_PunctuationChoice(String... puncts)
	{
		super(puncts);
	}
}

File: csrc/com/eagle/programmar/CMacro/Terminals/CMacro_PunctuationChoice.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// Purchase a Premium Edition license at:
// https://www.tangiblesoftwaresolutions.com/order/order-java-to-csharp.html
// ====================================================================================================

// Copyright Eagle Legacy Modernization, LLC, 2010-date
// Original author: Steven A. O'Hara, Dec 30, 2020

namespace com.eagle.programmar.CMacro.Terminals
{
	using TerminalPunctuationChoice = com.eagle.tokens.terminals.TerminalPunctuationChoice;

	public class CMacro_PunctuationChoice : TerminalPunctuationChoice
	{
		// Need default constructor for reading from the XML file
		public CMacro_PunctuationChoice() : base()
		{
		}

		public CMacro_PunctuationChoice(params string[] puncts) : base(puncts)
		{
		}
	}
}
